src/main/java/hudson/remoting/Channel.java

```diff
@@ -1618,7 +1665,9 @@ public OutputStream getUnderlyingOutput() {
      *      The remote port that the connection will be forwarded to.
      * @return
      *      Created {@link PortForwarder}
+     * @deprecated as of 3.39
```

Comment:
The `PortForwarder` class was Deprecated in #361, but no one bubbled up the deprecation within this module.
@jvz @jeffret-b